rust/protocol/src/crypto.rs

```diff
@@ -61,36 +73,53 @@ pub fn aes_256_cbc_decrypt(ctext: &[u8], key: &[u8], iv: &[u8]) -> Result<Vec<u8
         .map_err(|_| SignalProtocolError::InvalidCiphertext)?)
 }
 
-pub fn hmac_sha256(key: &[u8], input: &[u8]) -> Result<[u8; 32]> {
+/// The statically-known size of the output of [hmac_sha256].
+pub const HMAC_OUTPUT_SIZE: usize = 32;
```

Comment:
Arguably this is also bits-based (SHA-256 produces 256-bit hashes, which is 32 bytes.)

Reply:
Oh, yes! I have modified this to be `256 / BITS_PER_BYTE` as well.